CMakeLists.txt

```
@@ -14,8 +14,16 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wconversion -Wno-sign-conversion -Wpedantic -Wnon-virtual-dtor -Woverloaded-virtual)
 endif()
 
+option(FORCE_BUILD_VENDOR_PKG
```

@alsora nit: 
```suggestion
option(FORCE_BUILD_VENDORED_LIBYAML
```

or anything along those lines. `FORCE_BUILD_VENDOR_PKG` is a bit generic and could span several packages.

---

Having a generic name was done on purpose.
This is the same name as used in other vendor packages
https://github.com/ros2/spdlog_vendor/blob/master/CMakeLists.txt#L6
https://github.com/ros2/yaml_cpp_vendor/blob/master/CMakeLists.txt#L75
https://github.com/ros2/console_bridge_vendor/blob/master/CMakeLists.txt#L16

---

Good point. Nevermind.